FE/qr-menu/src/components/LoginView/Login.jsx

```diff
@@ -1,12 +1,92 @@
 import { Link } from 'react-router-dom';
 
-function Login() {
-  return (
-    <div>
-      <h1>TU ROBKO PRIDA LOGIN :)</h1>
-      <Link to="/menu/1">Click here for menu page!</Link>
-    </div>
-  );
+import Form from 'react-validation/build/form';
+import Input from 'react-validation/build/input';
+import CheckButton from 'react-validation/build/button';
+import React, { useState, useRef } from 'react';
+import { useQuery } from 'react-query';
+
+class NameForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {email: '',
+                  password: ''};
+
+    this.userResponseData = {accessToken: '',
+                        profile: {
+                        id: 0,
+                        fullname: ''
+                        }};
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    // Simple POST request with a JSON body using fetch
+    const requestOptions = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(this.state)
+    };
+    fetch('https://qrmenu-asdit.herokuapp.com/api/v1/authorization/login', requestOptions)
+      .then(res => {
+        if (res.ok) {
+          return res.json();
+        }
+        throw new Error("Login wasnt successful!");
+      })
+      .then(data => {
+        this.userResponseData.accessToken = data.accessToken;
+        this.userResponseData.profile.id = data.profile.id;
+        this.userResponseData.profile.fullname = data.profile.fullname;
+        console.log(this.userResponseData);
+      })
+      .catch(err => console.log(err));
+
+  }
```

Comment:
we don't want to have fetching inside component. take a look at @NicolasOrsag  PR